base/threadingconstructs.jl

```diff
@@ -75,7 +75,7 @@ end
 """
     Threads.@threads
 
-A macro to parallelize a for-loop to run with multiple threads. This spawns `nthreads()`
+A macro to parallelize a for-loop to run with multiple threads. This spawns [`nthreads()`](@ref)
```

Comment:
Is this going to work without the `Threads` specifier? Because `nthreads` is not exported. Not sure if this is tested on the buildbots...

Reply:
`make docs` worked locally...